content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Getting started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to get up and running with Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+To get started with Optimize, you will need the following:
+
+- A [Prisma Data Platform workspace](https://console.prisma.io).
+- A project that uses [Prisma Client](/orm/prisma-client) `5.16.1` or higher (we always recommend using the latest version of Prisma).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, or MongoDB database.
+
+## 1. Enable Optimize
+
+Login to your [Prisma Data Platform account](https://console.prisma.io) and then navigate to the [Optimize dashboard](https://optimize.prisma.io).
+
+## 2. Add Optimize to your application
+
+### 2.1. Update your `schema.prisma` file
+
+In your Prisma schema's `generator` block add the `tracing` preview feature:
+```prisma
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["tracing"]
+}
+```
+
+Then generate Prisma Client using:
+```terminal
+npx prisma generate
+```
+
+### 2.2. Install the Optimize Prisma Client extension
+
+Install the latest version of Prisma Client and Optimize Prisma Client extension:
+
+```terminal
+npm install @prisma/client@latest @prisma/extension-optimize
+```
+
+### 2.3. Extend your Prisma Client instance with the Optimize extension
+
+Add the following to extend your existing Prisma Client instance with the Optimize extension:
+
+```ts
+import { PrismaClient } from '@prisma/client'
+import { withOptimize } from '@prisma/extension-optimize'
+
+const prisma = new PrismaClient().$extends(withOptimize())
+```
+
+### 2.4. Use Prisma Optimize to generate insights
+
+Create a new recording in your Prisma Optimize dashboard to receive insights on your database queries by following these steps:
+1. Locate and click the **Start recording** button and then run your app. 
+2. Once you see query insights are generated for the desired queries or your app completes execution, locate and click the **Stop recording** button. You can then click on the individual queries to see more details.
+3. Navigate to the **Recommendations** tab to see if any improvements have been suggested by Optimize to improve query performance.
+    :::info
+    Use the [AI Explainer](/optimize/ai-explainer) feature to better understand recommendations and apply them within your Prisma model context.
+    :::
+You can also try out the [step-by-step example](https://github.com/prisma/prisma-examples/optimize/starter) to learn more. 
+
+## Need help?
+
+Reach out to us in the `#help-and-questions` channel on our [Discord](https://pris.ly/discord), or connect with our community to see how others are using Optimize.
```

Comment:
We should also specify that you can reach out to us by clicking on "Contact Support" button in the Optimize Dashboard.

This will open the Support form in which they can fill in the details.

Reply:
@nurul3101 I couldn't find the button, hence I skipped suggesting them this step.